Makefile

```diff
@@ -590,7 +590,9 @@ ifeq "$(INSTALL_OCAMLNAT)" "true"
 	  $(INSTALL_PROG) ocamlnat$(EXE) "$(INSTALL_BINDIR)"
 endif
 	cd "$(INSTALL_COMPLIBDIR)" && \
```

Comment:
Sorry for requesting a further change - please could you add a comment here noting that the libraries need to be passed individually?